src/decider/agent/DeciderAgent.php

```diff
@@ -543,4 +558,60 @@ protected function areOtherScannerFindingsAndRESOAgreed($licenseMatches)
     }
     return true;
   }
-}
+
+  private function getCopyrightsToDisableFalsePositivesClutter($uploadId, $clutter_flag)
+  {
+    if (empty($uploadId)) {
+      return array();
+    }
+    $agentName = 'copyright';
+    $uploadTreeTableName = $this->uploadDao->getUploadtreeTableName($uploadId);
+    $scanJobProxy = new ScanJobProxy($GLOBALS['container']->get('dao.agent'), $uploadId);
+    $scanJobProxy->createAgentStatus(array($agentName));
+    $selectedScanners = $scanJobProxy->getLatestSuccessfulAgentIds();
+    if (!array_key_exists($agentName, $selectedScanners)) {
+      return array();
+    }
+    $latestXpAgentId = $selectedScanners[$agentName];
+    $extrawhere = ' agent_fk='.$latestXpAgentId;
+    $allCopyrights = $this->copyrightDao->getScannerEntries('copyright',
+      $uploadTreeTableName, $uploadId, null, $extrawhere);
+
+    $copyrightJSON = json_encode($allCopyrights);
+    $tmpFile = tmpfile();
+    $tmpFilePath = stream_get_meta_data($tmpFile)['uri'];
+    fwrite($tmpFile, $copyrightJSON);
+    $deactivatedCopyrightData = $this->callCopyrightDeactivationClutterRemovalScript($tmpFilePath, $clutter_flag);
+    if (empty($deactivatedCopyrightData)) {
+      return array();
+    }
+    $deactivatedCopyrights = json_decode($deactivatedCopyrightData, true);
+    foreach ($deactivatedCopyrights as $deactivatedCopyright) {
+      $item = $deactivatedCopyright['uploadtree_pk'];
+      $itemTreeBounds = $this->uploadDao->getItemTreeBounds($item, $uploadTreeTableName);
+      $hash = $deactivatedCopyright['hash'];
+      $content = $deactivatedCopyright['content'];
+      $cpTable = 'copyright';
+      if ($deactivatedCopyright['is_copyright'] == "t") {
+        $action = '';
+        $hash = '';
+        $content = $deactivatedCopyright['edited_text'];
+      } else {
+        $hash = $deactivatedCopyright['hash'];
+        $action = 'delete';
+      }
+      $this->copyrightDao->updateTable($itemTreeBounds, $hash, $content, $this->userId, $cpTable, $action);
+      $this->heartbeat(1);
+    }
+    fclose($tmpFile);
+  }
+  private function callCopyrightDeactivationClutterRemovalScript($tmpFilePath, $clutter_flag)
+  {
+    $script = "copyrightDeactivationClutterRemovalScript.py";
+    $path_to_file =  __DIR__ . "/$script";
+    $command = "python3 " . $path_to_file . " -f" . $tmpFilePath . " -c" . $clutter_flag;
+    $output = shell_exec($command);
+    $this->heartbeat(0);
+    return $output;
+  }
+}
```

Comment:
Please add a new line at the end of the file.

And you can also add your copyright as this PR brings new changes to the class logic.